tensorflow/training/docker/2.5/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,283 @@
+FROM nvidia/cuda:11.2.1-base-ubuntu18.04
```

Comment:
it needs EFA to be included as its in TF2.4.1

Reply:
I already added EFA related commands from TF2.4.1. Am I missing something ?

Reply:
yes you need to add more changes

Reply:
Also missing package `jq`
https://github.com/aws/deep-learning-containers/blob/f9ee2faee183fa87049efb892648714e0d27fe68/tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu#L82

This package was installed in this PR : https://github.com/aws/deep-learning-containers/pull/1044

Reply:
done